subprojects/gst-plugins-bad/sys/msdk/gstmsdkallocator_libva.h

```diff
@@ -45,6 +45,14 @@ gboolean
 gst_msdk_export_dmabuf_to_vasurface (GstMsdkContext *context,
     GstVideoInfo *vinfo, gint fd, VASurfaceID *surface_id);
 
+VASurfaceID
+_get_va_surface (GstBuffer * buf, GstVideoInfo * info,
+    GstMsdkContext * msdk_context);
```

Comment:
do we need to expose this function in the header? Didn't see it called elsewhere

Reply:
> do we need to expose this function in the header? Didn't see it called elsewhere

I was thinking before that decoder may need to call this _get_va_surface. But currently it seems no need to export this help function. Thanks.